### PR TITLE
Fixing issue #649 giving different title to different updates open in…

### PR DESCRIPTION
**Pull request by farhaanbukhsh**
… different tabs

### CODE REVIEW

**Comment by ralphbean** [comment]
Looks good to me.  @lmacken, want to take a look?

File: bodhi/templates/master.html [hunk 1]
```diff
@@ -19,9 +19,11 @@
     <link rel="shortcut icon" href="${request.static_url('bodhi:static/ico/favicon.ico')}">
 
     <base href="${request.registry.settings['base_address']}"/>
-
-    <title>Fedora Updates System</title>
-
+    % if not update:
+    	<title>Fedora Updates System</title>
+    % else:
+    	<title>${update.title}</title>
```
Comment (lmacken) [inline bodhi/templates/master.html]: Maybe add `- Fedora Updates System` to the end to give the title some context?

File: bodhi/templates/master.html [hunk 2]
```diff
@@ -19,9 +19,11 @@
     <link rel="shortcut icon" href="${request.static_url('bodhi:static/ico/favicon.ico')}">
 
     <base href="${request.registry.settings['base_address']}"/>
-
-    <title>Fedora Updates System</title>
-
+    % if not update:
+    	<title>Fedora Updates System</title>
+    % else:
+    	<title>${update.title}-Fedora Updates System</title>
```
Comment (puiterwijk) [inline bodhi/templates/master.html]: I think it would be most clear to add spaces around the '-'.

**Comment by farhaanbukhsh** [comment]
Does it look good now ? :)

File: bodhi/templates/master.html (hunk 2 above)
Comment (farhaanbukhsh) [inline bodhi/templates/master.html]: True agreed!!

**Comment by farhaanbukhsh** [comment]
I think it looks good now :) @puiterwijk @lmacken @ralphbean

**Comment by puiterwijk** [comment]
Looks good to me. @lmacken?

**Comment by ralphbean** [comment]
Yup!  Thanks all!

**Comment by farhaanbukhsh** [comment]
:+1:  thanks a lot

**Pull request merged**